test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenTransactSpecs.java

```diff
@@ -496,6 +501,51 @@ public HapiApiSpec uniqueTokenTxnAccountBalance() {
 				);
 	}
 
+	public HapiApiSpec uniqueTokenTxnAccountBalancesForTreasury() {
```

Comment:
This test is not cleanly written. Adding this as comment for the 3rd time

Reply:
Test has been refactored